src/Microsoft.Extensions.Configuration.AzureAppConfiguration/StartupOptions.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    /// <summary>
+    /// Options used when initially loading data into the configuration provider.
+    /// </summary>
+    public class StartupOptions
+    {
+        /// <summary>
+        /// The maximum delay before timing out when loading data from Azure App Configuration on startup.
+        /// </summary>
+        public TimeSpan Timeout { get; set; } = TimeSpan.FromMinutes(1);
+
+        /// <summary>
+        /// The maximum delay between retries when loading data from Azure App Configuration on startup.
+        /// </summary>
+        public TimeSpan MaxRetryDelay { get; set; } = AzureAppConfigurationOptions.MaxRetryDelay;
```

Comment:
Is this to ignore what a user may set in ClientOptions? I don't think we need it. If a user set something, we can just honor it even if it may mess up the startup timeout.